health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/index.js

```
@@ -238,26 +235,23 @@ function DeliverySetup({ onSelect, config, formData, control, tabCount = 2, subT
+            if (activeDelivery) {
+              return {
+                ...i,
+                deliveries: i.deliveries.map((j) => ({
+                  ...j,
+                  deliveryRules: j.active ? action.payload.currentDeliveryRules : j.deliveryRules,
+                })),
+              };
+            }
+          }
+          return i;
         });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)